### ISSUE TITLE
The edge level will be overlaid on the circle when the edge is added after the starting point is rendered

### ISSUE THREAD

**Issue by hetao92** [antvis/G6#3132]
- [ ] I have searched the [issues](https://github.com/antvis/g6/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/bold-frost-z5bb5?file=/index.js)

### Steps to reproduce
1. add node 
```
graph.data({
  nodes: [
    {
      id: "1",
      style: {
        fill: "#FFC069"
      }
    }
  ],
  edges: []
});
graph.render();
```
2. add edge after the start node has been rendered
```
const _data = {
    nodes: [
      {
        id: "1",
        style: {
          fill: "#FFC069"
        }
      },
      {
        id: "2",
        style: {
          fill: "#FFC069"
        }
      }
    ],
    edges: [
      {
        source: "1",
        target: "2",
        id: "10"
      }
    ]
  };
  graph.changeData(_data);
```

| Environment | Info |
|---|---|
| g6 | 4.3.5 |
| System | - |
| Browser | - |

---
minimap also has this bug
![image](https://user-images.githubusercontent.com/18328704/129845072-2a0880da-7fc9-4883-af40-e6f3d510259b.png)
and if set node opacity, the node on main canvas, it can also been seen.

**Comment by Yanyan-Wang** [comment]
In your demo, the newly added edge is under the circle. It looks like over the circle since the circle has opacity: 0.3.

**Comment by hetao92** [comment]
> In your demo, the newly added edge is under the circle. It looks like over the circle since the circle has opacity: 0.3.

I think there are two questions.
1. I think setting opacity should not cause this problem. need to avoid it. could set index?
2. I had updated this demo and add minimap plugin, you can see this problem in minimap even if remove opacity.

thanks

**Comment by Yanyan-Wang** [comment]
the zIndex problem on the minimap is solved by v4.7.16
